docs/features/software-catalog/well-known-annotations.md

```diff
@@ -100,6 +100,28 @@ alongside the entity's source code, the value of this annotation can point to an
 absolute URL, matching the location reference string format outlined above, for
 example: `url:https://github.com/backstage/backstage/tree/master`
 
+### backstage.io/techdocs-skip-build
+
+```yaml
+# Example:
+metadata:
+  annotations:
+    backstage.io/techdocs-skip-build: 'true'
```

Comment:
Thinking about the naming of this: if I am an entity owner, with little or no knowledge of how backstage or the TechDocs plugin works, this could be unintuitive.  Does this somehow hook into my CI/CD system and cause it to skip building the docs?  Or?

I wonder if the annotation name should reflect _where_ the build takes place (as opposed to a binary skip/not skip)?  Like `backstage.io/techdocs-build-context: 'backstage'` (or `ci/cd` or...)

Has a nice, at least somewhat intuitive way to semantically describe the purpose of this annotation vs. the existing techdocs annotation:

- `backstage.io/techdocs-ref`: indicates that an entity has TechDocs at all, and if so, where the source of those docs exists
- `backstage.io/techdocs-build-context` (or whatever): indicates in what context those docs are built